docs/gitbook/tips/hadoop_tuning.md

```diff
@@ -75,13 +75,13 @@ feature_dimensions (2^24 by the default) * 4 bytes (float) * 2 (iff covariance i
 ```
 > 2^24 * 4 bytes * 2 * 1.2 ≈ 161MB
 
-When [SpaceEfficientDenseModel](https://github.com/apache/incubator-hivemall/blob/master/src/main/java/hivemall/io/SpaceEfficientDenseModel.java) is used, the formula changes as follows:
+When [SpaceEfficientDenseModel](https://github.com/myui/hivemall/blob/master/src/main/java/hivemall/io/SpaceEfficientDenseModel.java) is used, the formula changes as follows:
```

Comment:
`github.com/myui` is deprecated. 

Use https://github.com/apache/incubator-hivemall/blob/master/core/src/main/java/hivemall/model/SpaceEfficientDenseModel.java instead

other appearance of `github.com/myui` as well.

Reply:
I unintenntionay injected this modification. Is it OK to replace from `github.com/myui` into `apache/incubator-hivemall` if there is no redirect target? e.g. `bin/run_mixserv.sh` is still missing.

Reply:
OK, fixed github.com/myui as much as I can. https://github.com/apache/incubator-hivemall/pull/162/commits/774f75e8daa83ccd6295be903f08877dcd0c8f52